packages/opa-react/src/authz-provider.tsx

```diff
@@ -87,7 +83,9 @@ export default function AuthzProvider({
   defaultFromResult,
   retry = 3,
 }: AuthzProviderProps) {
-  const [queryClient] = useState(() => {
+  const queryClient = useMemo(() => {
```

Comment:
```suggestion
  const queryClient = useMemo<QueryClient | undefined>(() => {
```

Reply:
hm.. this is inferred so it's not needed.  I only mentioned this because you were casting below `as QueryClient`, then I would force existence `queryClient: queryClient!`

Reply:
Oh I wasn't aware that this was a possibility. Thanks! 💡